### PR TITLE
RDS Cluster Example Terraform - #66

### PR DESCRIPTION
**Pull request by robg-test**
For #66

### CODE REVIEW

**Review by RemakingEden — APPROVED** [review]

File: modules/aws/rds_serverless_cluster/examples/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  value = module.rds_cluster_setup.rds_cluster_instance_db_name
+}
+
+output "rds_cluster_replica_instance_endpoints" {
+  value = module.rds_cluster_setup.rds_cluster_replica_instance_endpoints
+}
+output "rds_cluster_replica_instance_ports" {
+  value = module.rds_cluster_setup.rds_cluster_replica_instance_ports
+}
+
```
Comment (cmbuckley) [inline modules/aws/rds_serverless_cluster/examples/outputs.tf:25]: avoid blank newlines

File: modules/aws/rds_serverless_cluster/main.tf [hunk 2]
```diff
@@ -112,3 +111,5 @@ resource "random_password" "password" {
 resource "random_id" "secrets_id" {
   byte_length = 8
 }
+
```
Comment (cmbuckley) [inline modules/aws/rds_serverless_cluster/main.tf:114]: avoid blank newlines. I would suggest we use an editorconfig

Comment (robg-test) [reply modules/aws/rds_serverless_cluster/main.tf:114]: Is that VSCode thingy? (Jetbrains Fiend)

Could do a prehook to TF fmt, as that's relatively development platform agnostic. (Seperate issue though with that one)

Comment (cmbuckley) [reply modules/aws/rds_serverless_cluster/main.tf:114]: [editorconfig](https://editorconfig.org/) is an IDE-agnostic way of defining coding styles for a project. However, it seems enforcing a _single_ trailing newline isn't something it covers yet: https://github.com/editorconfig/editorconfig/issues/269

A text file should end with **exactly one** newline - you can [enforce a newline at the end of a file in your editor](https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file), but terraform fmt is not opinionated about vertical whitespace.

Comment (robg-test) [reply modules/aws/rds_serverless_cluster/main.tf:114]: I've switched it on in Webstorm 
image

Don't know about other IDE's

Comment (robg-test) [reply modules/aws/rds_serverless_cluster/main.tf:114]: Dont ask why I use webstorm for this.
It's a mystery lost to time

Comment (cmbuckley) [reply modules/aws/rds_serverless_cluster/main.tf]: Make sure you tick that that one too - all files should end with a line break

**Pull request merged**